modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -12,6 +12,16 @@
 		<td>//input[@placeholder="Choose an Action"]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_ERROR_MESSAGE_VALIDATION</td>
+		<td>//div[contains(@class,'form-group')]//label[text()='Message']/..//input[@type='text']</td>
+		<td></td>
+	</tr>
```

Comment:
Are you sure there is no path to this step? Maybe exist a path that could be used here into Objects.